static/src/js/components/AccessMethod/__tests__/AccessMethod.test.js

```diff
@@ -1510,4 +1510,171 @@ describe('AccessMethod component', () => {
+        method: {
+          swodlr: {
+            swodlrData: {
+              params: {
+                rasterResolution: 90,
+                outputSamplingGridType: 'UTM',
+                outputGranuleExtentFlag: true
+              },
+              custom_params: {}
+            }
+          }
+        }
+      })
+    })
+
+    test('selecting a LAT/LON sample grid calls onUpdateAccessMethod with automatic rasterResolution value adjustment', async () => {
+      const user = userEvent.setup()
+
+      const collectionId = 'collectionId'
+      const { onUpdateAccessMethod } = setup({
+        accessMethods: {
+          swodlr: {
+            type: 'SWODLR',
+            supportsSwodlr: true
+          }
+        },
+        metadata: {
+          conceptId: collectionId
+        },
+        selectedAccessMethod: 'swodlr'
+      })
+
+      const latLonCheckbox = screen.getByRole('radio', { name: 'LAT/LON' })
+
+      await user.click(latLonCheckbox)
+
+      expect(onUpdateAccessMethod).toHaveBeenCalledTimes(1)
+      expect(onUpdateAccessMethod).toHaveBeenCalledWith({
+        collectionId: 'collectionId',
+        method: {
+          swodlr: {
+            swodlrData: {
+              params: {
+                rasterResolution: 3,
+                outputSamplingGridType: 'GEO',
+                outputGranuleExtentFlag: false
+              },
+              custom_params: {}
+            }
+          }
+        }
+      })
+    })
+
+    test('can update individual granules MGRS band and UTM Zone Adjust', async () => {
+      const user = userEvent.setup()
+
+      const collectionId = 'collectionId'
+      const { onUpdateAccessMethod } = setup({
+        accessMethods: {
+          swodlr: {
+            type: 'SWODLR',
+            supportsSwodlr: true
+          }
+        },
+        metadata: {
+          conceptId: collectionId
+        },
+        selectedAccessMethod: 'swodlr',
+        granuleMetadata: {
+          'G1261369123-POCLOUD': {
+            value: 'test',
+            id: 'G1261369123-POCLOUD'
+          },
+          'G1261369376-POCLOUD': {
+            value: 'test',
+            id: 'G1261369376-POCLOUD'
+          }
+        },
+        projectCollection: {
+          granules: {
+            addedGranuleIds: [
+              'G1261369123-POCLOUD',
+              'G1261369376-POCLOUD'
+            ]
+          }
+        }
+      })
+
+      const advancedOptionsToggleButton = screen.getByTestId('advancedOptionsToggle')
+      await user.click(advancedOptionsToggleButton)
+
+      const firstGranuleUTMZonePlusOne = screen.getByTestId('G1261369123-POCLOUD-plus-1-UTM-zone')
+      user.click(firstGranuleUTMZonePlusOne)
+
+      waitFor(() => expect(onUpdateAccessMethod).toHaveBeenCalledTimes(1))
+
+      waitFor(() => expect(onUpdateAccessMethod).toHaveBeenCalledWith({
+        collectionId: 'collectionId',
+        method: {
+          swodlr: {
+            json_data: {
```

Comment:
I think this might be off not sure how its passing but, isn't `json_data` no longer being used for this?

Reply:
Fixed test